src/components/Palette/__stories__/Palette.stories.tsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+
+import * as icons from '@gravity-ui/icons';
```

Comment:
Let's import only a few for example, not need to import all

Reply:
Done